v2/tools/generator/internal/reporting/type_catalog_report.go

```diff
@@ -390,3 +432,11 @@ func (tcr *TypeCatalogReport) inPackage(
 
 	return result
 }
+
+func (tcr *TypeCatalogReport) asCount(value int, singular string, plural string) string {
```

Comment:
minor, naming: This feels more like `formatCount` or something to me? As it's purely a string formatting method.

Reply:
Renamed.